transport/controller.cc

```diff
@@ -65,6 +66,124 @@ future<> controller::start_server() {
     return do_start_server().finally([this] { _ops_sem.signal(); });
 }
 
+// Wrapper for cql_server::listen to invoke it on all shards and log address and listen options.
+static future<> listen(sharded<cql_server>& cserver, socket_address addr, std::shared_ptr<seastar::tls::credentials_builder> creds, bool is_shard_aware, bool keepalive, std::optional<file_permissions> unix_domain_socket_permissions) {
+    co_await cserver.invoke_on_all([addr, creds, is_shard_aware, keepalive, unix_domain_socket_permissions] (cql_server& server) {
+        return server.listen(addr, creds, is_shard_aware, keepalive, unix_domain_socket_permissions);
+    });
+
+    logger.info("Starting listening for CQL clients on {} ({}, {})"
+            , addr, creds ? "encrypted" : "unencrypted", is_shard_aware ? "shard-aware" : "non-shard-aware"
```

Comment:
Consider changing to something shorter like:
```suggestion
    logger.info("Starting listening for CQL clients on {} (encrypted: {}, shard-aware: {})", addr, creds, is_shard_aware)
```
(or better print "true" or "false")

Reply:
it's code already existing on master, I'd not change it in this PR